FunctionalTableData/CellStyle.swift

```diff
@@ -34,9 +49,9 @@ public struct CellStyle {
 	/// You use these constants when setting the value of the [accessoryType](apple-reference-documentation://hspQPOCGHb) property.
 	///
 	/// Supported by `UITableView` only.
-	public var accessoryType: UITableViewCell.AccessoryType = .none
+	public var accessoryType: UITableViewCell.AccessoryType
 	/// The view's selection color.
-	public var selectionColor: UIColor? = CellStyle.selectionColor
```

Comment:
These property-level defaults were never used, since the public `init` touches every property and overrides them with its parameter defaults.